app/views/admin/ride_uploads/_form.html.haml

```diff
@@ -0,0 +1,23 @@
+= form_for( @ride_upload, url: admin_ride_zone_ride_uploads_path(@ride_zone)) do |f|
+  - if @ride_upload.errors.any?
+    #error_explanation
+      %h2= "#{pluralize(@ride_upload.errors.count, "error")} prohibited this ride_upload from being saved:"
+      %ul
+        - @ride_upload.errors.full_messages.each do |message|
+          %li= message
+
+  .col-sm-8.field
+    = f.label :name
+    = f.text_field :name, autofocus: true, class: 'form-control'
+    
+  .col-sm-8.field
+    = f.label :description
+    = f.text_area :description, class: 'form-control'
+    
+  .col-sm-8.field
+    = f.label :file
+    = f.file_field :csv, class: 'form-control'
+    
+    
+  .col-sm-8.actions
+    = f.submit t(:save), class: 'btn btn-primary', disable_with: "#{t(:saving)}..."
```

Comment:
Is the `#` a typo?

Reply:
Also `...` (three dots) should be `…` (ellipses) for screen readers

Reply:
to get the substitution with the ellipsis after it requires #{}, I think? adding a proper ellipsis/

Reply:
The screen width I happened to be at put `#` on one line and `{…}` on the next, so it wasn't clear that it was string interpolation. This looks perfect now.